api/src/exceptions/invalid-config.ts

```diff
@@ -0,0 +1,7 @@
+import { BaseException } from '@directus/shared/exceptions';
+
+export class InvalidConfigException extends BaseException {
+	constructor(message = 'Invalid config', provider?: string) {
```

Comment:
Lets not lock this to a "provider" string flag, but instead just pass on the a full object of extensions (seeng "Invalid Config" could be used for anything)